pkg/apis/sources/v1alpha1/awscloudwatch_types.go

```diff
@@ -46,6 +46,10 @@ var (
 
 // AWSCloudWatchSourceSpec defines the desired state of the event source.
 type AWSCloudWatchSourceSpec struct {
+	// Adapter spec overrides parameters.
+	// +optional
+	AdapterOverrides *v1alpha1.AdapterOverrides `json:"adapterOverrides,omitempty"`
```

Comment:
nit: (applies to all Source types)
All `flow`, `extensions` and `router` types have this at the _end_ of their spec. I think it makes a lot more sense that way in terms of readability (at the end) because the type-specific attributes remain at the beginning.
Also, it's nice to be consistent in the way we do things, as a general good practice.

Reply:
I put this field at the beginning of the structure because there is another common source field `duckv1.SourceSpec` set as a first-line. Also, some of them seemed to be alphabetically ordered. Anyways, I have moved it to the bottom.